consensus/core/src/authority_service.rs

```diff
@@ -242,9 +253,27 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
         // For now ask dag state directly
         let blocks = self.dag_state.read().get_blocks(&block_refs);
 
-        // Return the serialised blocks
+        // Now check if an ancestor's round is higher than the one that the peer has. If yes, then serve
+        // that ancestor blocks up to `MAX_ADDITIONAL_BLOCKS`.
+        let mut ancestor_blocks = vec![];
+        if !highest_accepted_rounds.is_empty() {
```

Comment:
nit: this check is not necessary because of the check earlier & committee size can never be 0